Makefile

```diff
@@ -82,6 +82,11 @@ update-dependencies:
 	@make freeze-dependencies
 
 
+.PHONY: format
+format: setup
+	@. .venv/bin/activate && black .
```

Comment:
Decided to call the target `format`, and not `black`, as there might be other formatting tools we might want to add (JS/HTML/CSS..?).